server/src/main/java/org/elasticsearch/action/get/MultiGetResponse.java

```diff
@@ -204,22 +204,24 @@ private static MultiGetItemResponse parseItem(XContentParser parser) throws IOEx
             switch (token) {
                 case FIELD_NAME:
                     currentFieldName = parser.currentName();
-                    if (INDEX.match(currentFieldName) == false && TYPE.match(currentFieldName) == false &&
-                            ID.match(currentFieldName) == false && ERROR.match(currentFieldName) == false) {
+                    if (INDEX.match(currentFieldName, parser.getDeprecationHandler()) == false &&
+                        TYPE.match(currentFieldName, parser.getDeprecationHandler()) == false &&
```

Comment:
Same deal as last time with indentation. Also, why not `&& TYPE.match` like you usually do? I like that.